Utils/request_contributor_review.py

```diff
@@ -14,7 +14,7 @@
 PACK_METADATA = "pack_metadata.json"
 XSOAR_SUPPORT = "xsoar"
 PACK_METADATA_GITHUB_USER_FIELD = "githubUser"
-PR_COMMENT_PREFIX = "### Your contributed pack has been modified. Please review the changes.\n"
+PR_COMMENT_PREFIX = "pack has been modified on files:\n"
```

Comment:
No need for this constant

Reply:
It is also used to check if a message has already been sent
So it should stay constant so that every change should be in both places

Reply:
👍